pyquil/tests/test_operator_estimation.py

```diff
@@ -723,7 +721,7 @@ def test_bit_flip_channel_fidelity(forest, use_seed):
     p.define_noisy_gate("I", [0], kraus_ops)
 
     # prepare TomographyExperiment
```

Comment:
```suggestion
    # prepare Experiment
```

bunch of these below